docs/data/charts/overview/overview.md

```diff
@@ -7,34 +7,29 @@ packageName: '@mui/x-charts'
 
 # MUI X Charts
 
-<p class="description">A fast and extendable library of react chart components for data visualization.</p>
+<p class="description">A collection of React chart components for data visualization.</p>
 
 {{"component": "@mui/docs/ComponentLinkHeader", "design": false}}
 
 ## Overview
 
-The `@mui/x-charts` is an MIT library for rendering charts relying on [D3.js](https://d3js.org/) for data manipulation and SVG for rendering.
-And, like other MUI X components, charts are production-ready components that integrate smoothly into your app.
+MUI X Charts is a library of production-ready components for rendering charts with React.
+It uses [D3.js](https://d3js.org/) for data manipulation and SVGs for rendering.
 
-With a high level of customization, MUI X Charts provides three levels of customization layers: **single components** with great defaults, extensive **configuration props**, and **subcomponents** for flexible composition.
-Additionally, you can also use all the [MUI System](https://mui.com/system/getting-started/) tools, such as the theme override or the `sx` prop.
+The components provide a high level of customization, with beautiful Material Design-based defaults as well as extensive configuration props and flexible composition options.
+They also have access to all [MUI System](https://mui.com/system/getting-started/) tools such as theme overrides and the `sx` prop.
 
 {{"demo": "ChartsOverviewDemo.js", "defaultCodeOpen": true}}
 
-## Using the documentation
+## Using this documentation
 
-The MUI X Charts documentation has a slightly different structure than other MUI X components.
-Instead of having a long page for each, the pages are divided in two:
+Each Chart type has two accompanying documents:
 
-1. General description of the built-in features the component provides.
-2. A set of examples demonstrating the component with customizations.
+1. **Overview** – a general description of built-in features
+2. **Demo** – a collection of custom examples
+
+See the **Common features** section for details on features that are shared across all Charts.
 
 ## All MUI X Charts components
 
 {{"component": "modules/components/ChartComponentsGrid.js"}}
-
-## Supported features
-
-The features that are shared across multiple Charts components, such as axes and legends, are also documented on separate pages.
-
-{{"component": "modules/components/ChartFeaturesGrid.js"}}
```

Comment:
Are we sure about removing this section/component? 🤔

Reply:
Is it missing something else than `Highlighting`, which has recently been split off from `Tooltip & Highlights`? 🤔 
We can remove it if you feel it doesn't bring enough value and @alexfauquette agrees, but I'd be up for removing the component as well. 👍

Reply:
I see 10 total "Common features" in the sidebar nav, which seems to be what this section is drawing attention to. It's kind of nice to have but we don't have anything like it for any of the other packages.

Reply:
I agree. This was one of the last creations from Danilo. Maybe he had ideas of adding similar element to each product documentation. 🤔 🤷

Reply:
@noraleonte @zanivan what do you think about this features "demo"? Should we keep it and add one to the other Overview pages, or remove it here?

Reply:
For me this a way to attract the attention of a new user to the "Common features".

material-ui devs are used to to structure 1 page = 1 component. This section is to highlight that we also have dedicated pages to the common features. The demo is mostly here to take some visual space.

![image](https://github.com/user-attachments/assets/389bdf35-6ebb-4213-b4b1-3c3631203d90)

Since it's part of the overview page. It does not need to be up to date. I don't expect developers to got on the overview page to find those docs. It's more a call to action to discover docs pages.

Reply:
Got it, well if we're happy with it as is then I'm ok with keeping it at least until this Overview page gets a full design makeover.

Reply:
In the meantime, it shouldn't be difficult to add the missing items. If you agree, let me know which items are missing, and I can open a PR to address this after the current PR is merged :)

Reply:
Much appreciated @zanivan ! Here's the list of all the Charts features from the nav bar:

Axis
Custom components
Composition
Label
Legend
Stacking
Styling
Tooltip
Highlighting
Zoom and pan
Preview

Reply:
@zanivan I went ahead and updated this demo here, let me know what you think!

Reply:
We could potentially remove "Custom components" and "Composition" from the grid if we want to make it easier to maintain a full rectangle grid with 8 items vs 10.